pyk/Makefile

```diff
@@ -30,24 +30,24 @@ poetry-install:
 
 POETRY_RUN := poetry run
 
-# Tests
 
-PYK_TEST_ARGS :=
+# Tests
 
 test: test-unit test-integration test-pyk test-kit
 
 test-unit: poetry-install
-	$(POETRY_RUN) python -m unittest discover tests --failfast --verbose $(PYK_TEST_ARGS)
+	$(POETRY_RUN) python -m unittest discover tests --failfast --verbose
 
```

Comment:
We need a way to run an individual test, otherwise the turnaround time is too long. I was using this as to do `make test-intergration PYK_TEST_ARGS='-k defn_test'` for example, to just run the tests in defn_tests.

If you add a different way to run an individual test with a single short command (not having to manually activate virtual env or poetry), then that's fine, otherwise let's add this back.

Reply:
I see, this is a nice trick. I'll add it back.